Integrations/integration-Zendesk.yml

```diff
@@ -430,10 +437,16 @@ script:
         var md = '### Zendesk ticket #' + resDetails.ticket.id + " - " + resDetails.ticket.subject + "\n";
         var ec = {}
         var usersEc = {}
+        var org = {};
         var newContext = {};
 
         md += 'Requester: __' + getUserName(resDetails.ticket.requester_id) + '__\n';
         usersEc[resDetails.ticket.requester_id]=getUserName(resDetails.ticket.requester_id);
+        if (resDetails.ticket.organization_id) {
+            org = getOrganizationDetails(resDetails.ticket.organization_id);
+            md += 'Organization: __' + org.name + '__ (' + org.tags.join(', ') +')\n';
```

Comment:
please share screenshot of this org results, not sure I got how will it look like.

Also, will you always have `org.tags`? can it be null somehow? If so, just do:
`(org.tags || []).join(', ')`  - will handle this

Reply:
will fix.
With tags:
![image](https://user-images.githubusercontent.com/1042046/34338364-b13ea3b0-e970-11e7-9cf1-214eeaa8927b.png)

without tags:
![image](https://user-images.githubusercontent.com/1042046/34338382-e802a4a0-e970-11e7-9b91-48073ecec411.png)